modules/op-release-notes-1-6.adoc

```diff
@@ -0,0 +1,194 @@
++
+[IMPORTANT]
+====
+Breaking change: The `podTemplate` field, which was deprecated as part of link:https://github.com/tektoncd/triggers/pull/1100[#1100], has been removed.
+====
+
+* The current release removes the deprecated `replicas` field from the `EventListener` specification. (link:https://github.com/tektoncd/triggers/pull/1113[#1113])
++
+[IMPORTANT]
+====
+Breaking change: The deprecated `replicas` field has been removed.
+====
+
+[id="removing-home-workingdir-values-1-6_{context}"]
+=== Removing the HOME="/tekton/home" and workingDir="/workspace" values from Steps
+// TBD - Where does this release note belong?
+
+In {pipelines-title} 1.6, the values of `HOME="/tekton/home"` and `workingDir="/workspace"` are removed from the `Steps` specification.
+
+Instead, {pipelines-title} sets `HOME` and `workingDir` to the values defined by your Steps containers. You can override these values in your `Steps` YAML file.
+
+To get the older behavior, you can change the following fields in `TektonConfig` to `false`. Their default values are `true`.
+
+[source,yaml]
+----
+spec:
+  pipeline:
+    disable-working-directory-overwrite: false
+    disable-home-env-overwrite: false
+----
+
+[IMPORTANT]
+====
+The feature flags are temporarily available. These flags are candidates for complete removal from a future minor release of the upstream product at least nine months from when this release goes live.
+====
+
+(link:https://issues.redhat.com/browse/SRVKP-1465[SRVKP-1465])
+
+[id="known-issues-1-6_{context}"]
+== Known issues
+
+* When you run Maven and Jib-Maven cluster tasks on IBM Power Systems (ppc64le), IBM Z, and LinuxONE (s390x) clusters, set the `MAVEN_IMAGE` parameter value to `maven:3.6.3-adoptopenjdk-11`.
+// issue # is unknown.
+
+* On IBM Power Systems, IBM Z, and LinuxONE, the `s2i-dotnet` cluster task is unsupported.
+// issue # is unknown.
+
+* Before you install catalog tasks on IBM Power Systems, IBM Z, and LinuxONE using `tkn hub`, make sure to check task information about these platforms. For example, using the following command: `tkn hub info task <name>`
+// issue # is unknown.
+
+
+[id="fixed-issues-1-6_{context}"]
+== Fixed issues
+
+The `tkn hub` command is now supported on IBM Power Systems, IBM Z, and LinuxONE.
+// issue # is unknown.
+
+=== CLI 0.21.0 release
+
+* Before this update, the terminal was not available after the user ran a `tkn` command, and the pipeline run was done, even if `retries` were specified. Specifying a timeout in the task run or pipeline run had no effect. This update fixes the issue so that the terminal is available after running the command. (link:https://github.com/tektoncd/cli/issues/1459[#1459])
+
+=== CLI 0.20.0 release
+
+* Before this update, using the `tkn version --component=<component>` command did not return the component version. This update fixes the issue so that you can get the component version by using the `--component` option. (https://github.com/tektoncd/cli/pull/1408[#1408])
```

Comment:
```suggestion
* Before this update, using the `tkn version --component=<component>` command did not return the component version. This update fixes the issue so that you can get the component version. (https://github.com/tektoncd/cli/pull/1408[#1408])
```
I think it's unnecessary to mention the name of the flag again

Reply:
Updated. Thanks.